### DISCUSSION TITLE
[HOOK | IMPROVEMENT] Add a actionWebserviceObjectAdd hook

### DISCUSSION THREAD

**Discussion by BlackKerio** [PrestaShop/PrestaShop#33397] [Ideas]
I would like to ask you to add a hook to monitor the webservice API. Currently some hooks depend only on frontoffice and backoffice.

Insert one hook into the WebserviceRequestCore class with object parameter and sent xml file for any additional information

It is not necessarily used to override the function

Example: when creating a new customer using the webservice, the customer is not informed by email.

**Comment by khouloudbelguith** [comment]
Hi @BlackKerio,

Thanks for your report.
Ping @PrestaShop/prestashop-product-team @PrestaShop/prestashop-core-developers what do you think?

Thanks!
